src/index.js

```diff
@@ -355,6 +356,22 @@ export class DownloaderHelper extends EventEmitter {
             const _fileName = this.__getFileNameFromHeaders(response.headers);
             this.__filePath = this.__getFilePath(_fileName);
             this.__fileName = this.__filePath.split(path.sep).pop();
+            if (fs.existsSync(this.__filePath)) {
+                const downloadedSize = this.__getFilesizeInBytes(this.__filePath);
+                if (typeof this.__opts.override === 'object' &&
+                    this.__opts.override.skip && (
+                    this.__opts.override.skipSmaller ||
+                    downloadedSize >= this.__total)) {
+                    this.emit('skip', {
+                        totalSize: this.__total,
+                        fileName: this.__fileName,
+                        filePath: this.__filePath,
+                        downloadedSize: downloadedSize
+                    });
+                    this.state = this.__states.SKIPPED;
```

Comment:
this needs to be changed to `this.__setState(this.__states.SKIPPED);` in order to sends the emit too, but do not worry i will change it for you , since i will also push some changed i already had on my dev branch for this version